ansible/roles/fanout/templates/sonic_deploy_202012.j2

```diff
@@ -116,6 +116,23 @@
         }
     },
 
+{% if 'mellanox' in fanout_sonic_version["asic_type"] %}
+    "BUFFER_PG": {
+    {% for port_name in fanout_port_config %}
+        "{{ port_name }}|0-7": {
+            "profile": "[BUFFER_PROFILE|ingress_lossy_profile]"
+        }{% if not loop.last %},{% endif %}
+    {% endfor %}
+    },
+    "BUFFER_QUEUE": {
+    {% for port_name in fanout_port_config %}
+        "{{ port_name }}|0-7": {
+            "profile": "[BUFFER_PROFILE|q_lossy_profile]"
+        }{% if not loop.last %},{% endif %}
+    {% endfor %}
+    },
+{% endif %}
+
```

Comment:
We need to add the profile into config_db.json

Reply:
Thanks, added `BUFFER_POOL` and `BUFFER_PROFILE`.